src/platforms/ios/iaphandler.mm

```diff
@@ -381,17 +381,17 @@ - (void)request:(SKRequest*)request didFailWithError:(NSError*)error {
 
   connect(request, &NetworkRequest::requestCompleted,
           [this, ids](QNetworkReply*, const QByteArray&) {
-    logger.log() << "Purchase request completed";
-    SettingsHolder::instance()->addSubscriptionTransactions(ids);
+            logger.log() << "Purchase request completed";
```

Comment:
Had a gross merge and this ended up being included.

Reply:
Can you revert these changes?